src/constants/memory/wram.asm

```diff
@@ -17,7 +17,17 @@ section "WRAM Bank0", wram0[$c000]
 
 wram0Section EQU $C000
 
-; Buffer Area for 12 Link OAM tiles. Copied to OAM by DMA.
+; Generic 40 tiles buffer for OAM data. Copied to OAM by DMA.
+; During World gameplay, the OAM buffer is split between 12 OAM
+; tiles for Link, and 28 OAM tiles for dynamic entities.
+; Structure:
+;  byte 0: Y position
+;  byte 1: X position
+;  byte 2: tile n°
+;  byte 3: attribute
+wOAMBuffer:: ; C000
```

Comment:
remove the address after the label.
Addresses are noted after the ds X ;

Reply:
Good catch 👍 

Fixed.